README.md

```diff
@@ -1,3 +1,26 @@
-# Fullstack Authentication Example with Next.js and NextAuth.js
+# Court-Share
+
+
+
+## Installation
+
+1. Clone the repository.
+2. Install dependencies with `npm install`.
+3. Install docker `https://docs.docker.com/get-docker/`
+
+## Running App locally
+1. Create a `.env`, add `URL_PRISMA` with an string value of `postgresql://postgres:password@localhost:5432/court-share?schema=public`
+2. In the prisma/schema.prisma file, edit datasource db block to this:
+```
+datasource db {
+  provider   = "postgresql"
+  url        = env("URL_PRISMA") // uses connection pooling
+  extensions = [postgis]
+}
+```
+3. start the docker container with `docker-compose up -d`
+4. Run `npx prisma migrate dev --name init`
+5. Run `npx prisma studio`
+
+
```

Comment:
Let's add a sub section that instructs the users on how to run the actual server, `npm run dev` or etc.

Reply:
As per our conversation yesterday, I updated the README file to match the package.json script command.